src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -159,7 +159,9 @@
                 "all_or_nothing": "Alles-oder-Nichts",
                 "proportional_with_penalty": "Anteilig mit Abzügen"
             },
-            "importJSON": "Fragen importieren (JSON)"
+            "importJSON": "Fragen importieren (JSON)",
+            "explanationAnswered": "Frage wurde zur Kenntnis genommen",
```

Comment:
This is not a good translation, please use "Du hast die Frage beantwortet" bzw. "Du hast die Frage noch nicht beantwortet"